pkg/graveler/errors.go

```diff
@@ -24,6 +24,7 @@ var (
 	ErrInvalidMergeBase             = fmt.Errorf("only 2 commits allowed in FindMergeBase: %w", ErrInvalidValue)
 	ErrNoCommitGeneration           = errors.New("no commit generation")
 	ErrNoMergeBase                  = errors.New("no merge base")
+	ErrInvalidMergeStrategy         = errors.New("invalid merge strategy")
```

Comment:
If this error actually reaches the user to be returned by HTTP, we should strip accumulated messages from it on the HTTP response.  So consider
```suggestion
	ErrInvalidMergeStrategy         = wrapError(ErrUserVisible, "invalid merge strategy")
```